compute_endpoint/tests/unit/test_status_reporting.py

```diff
@@ -36,8 +36,9 @@ def test_status_reporting(
 
     # Confirm heartbeats in regular intervals
     for _i in range(3):
-        message = results_q.get(timeout=0.2)
-        assert isinstance(message, bytes)
+        q_msg = results_q.get(timeout=1)
```

Comment:
Was this sometimes failing with 0.2 timeout?

Reply:
No, but I modified that (perhaps incorrectly) to be in line with a number of other timeouts&nbsp;&mdash;&nbsp;I'm trying to avoid flaky tests and have noted that the infrastructure afforded us by GH (for free) is weak at best.  I figure that the `.get()` will return as soon as the underlying queue has a value, so 0.2 is effectively the same as 1 in the happy-path.  On a "slow day," I'm assuming that 1s is beyond overkill, so if we've a bug and die 0.8s slower than previously, "NBD."  Meanwhile, at 1s, I figure also that we'll pay more notice to the failure at 1s vs 0.2s for precisely the argument above: "Ah, it's just GH's weak infrastructure."

But the biggest motivator here was congruence with other tests and wanting to avoid false failure.